### ISSUE TITLE
Add NGOSource ED Badge to "about" or footer of OLS homepage

### ISSUE THREAD

**Issue by yochannah** [open-life-science/open-life-science.github.io#929]
This tells organisations that our work is equivalent to a USA 501(c)3. 

> As an organization with a certification from NGOsource, you are also eligible to use the NGOsource badge image on your website to let others know about your status. Please find the badge image here: https://techsoup.app.box.com/file/1538547383706?v=NGOsourceFileBadgeImage. To use the badge on your website, please (1) Embed the image wherever you would like it to appear, and (2) Make sure the badge image links to this webpage: https://www.ngosource.org/about-equivalency-determination-on-file-badge.

**Comment by NPDebs** [comment]
Thanks, @yochannah. Acknowledging this while also asking if there's a deadline for this task.

**Comment by yochannah** [comment]
nope, BUT the equivalency determination is only valid for one year, sooooo....
